### PR TITLE
Avoid mentioning users again on a recurring basis for the commits they had been reminded before

### PR DESCRIPTION
**Pull request by vishaltelangre**
### Why?

We don’t want to ping people again because maybe there are blockers in getting their commits verified.

### Screenshots

image

### CODE REVIEW

File: app/models/commit.rb [hunk 1]
```diff
@@ -15,6 +15,8 @@ class Commit < ActiveRecord::Base
   scope :deployed, -> { where(deploy_status: "succeeded") }
   scope :deployed_but_not_verified, -> { deployed.where("commits.state NOT IN (?) OR commits.state IS NULL", ['failed', 'verified']) }
   scope :deployed_but_not_verified_for_duration, ->(duration) { deployed_but_not_verified.where("commits.updated_at < ?", duration.ago) }
+  scope :yet_to_be_reminded, -> { where(notified_at: nil) }
```
Comment (vishaltelangre) [inline app/models/commit.rb:18]: Utilizing the unused `notified_at` column to ensure that we don't send notification for the same commit over again.

**Comment by vishaltelangre** [comment]
@sharang-d _a Please review.

**Comment by sharang-d** [comment]
@chrismaximin _a

**Review by chrismaximin — APPROVED** [review]
@vishaltelangre _a

**Pull request merged**